lib/Qi.pm

```diff
@@ -10,7 +14,9 @@ sub startup {
   my $config = $self->plugin('Config');
 
   #Model
-   $self->helper(pg => sub { state $pg = Mojo::Pg->new(shift->config('pg')) });
+  $self->helper(pg => sub { state $pg = Mojo::Pg->new(shift->config('pg')) });
+  $self->helper(sql => sub{ state $sql = SQL::Abstract->new});
+  $self->helper(base => sub { state $base = Qi::Model::Base->new(pg => shift->pg , sql => $self->sql) });
```

Comment:
We talked about the base class the other day...  What you want to do with a base class is sub class it.  So, you'd have your base class (CRUD) that all of your models would subclass and then you'd put any model-specific methods inside of that.  You don't want to use a base class directly; a model will subclass (extend) a base class and you'll use that subclass.